### PR TITLE
7.x.labels

### PR DESCRIPTION
**Pull request by willtp87**

### CODE REVIEW

File: islandora_usage_stats.module [hunk 1]
```diff
@@ -312,18 +312,18 @@ function islandora_usage_stats_download_ds_form($pid, $dsid) {
...
     if ($file) {
-//         drupal_add_http_header('Content-type', 'application/x-unknown; utf-8');
-        drupal_add_http_header('Content-type', $mime_type);
-        drupal_add_http_header('Content-Disposition', 'attachment; filename' . $sanitized_label);
-        echo $file;
-        exit();
-    } else {
-        drupal_set_message(t('Specified file does not exist!'));
+      drupal_add_http_header('Content-type', $mime_type);
+      drupal_add_http_header('Content-Disposition', 'attachment; filename' . $sanitized_label);
```
Comment (adam-vessey) [inline islandora_usage_stats.module:321]: Making a note: This specification of `Content-Disposition` doesn't appear to be valid? Should be `attachment; filename="whatever.ext"`, I think? The HTTP spec: http://www.w3.org/Protocols/rfc2616/rfc2616-sec19.html#sec19.5.1

File: views/islandora_usage_stats.views.inc [hunk 2]
```diff
@@ -107,28 +176,26 @@ function islandora_usage_stats_views_data() {
...
       ),
       'sort' => array(
         'handler' => 'views_handler_sort',
       ),
       'filter' => array(
-        'handler' => 'views_handler_filter_string',
-      ),
-      'argument' => array(
-        'handler' => 'views_handler_argument_string',
+        'handler' => 'views_handler_filter_numeric',
```
Comment (adam-vessey) [inline views/islandora_usage_stats.views.inc:198]: Not... 100% sure that this really makes sense, to have the IDs be "numeric"?... Yes, I realize they're integers, but being "numeric" seems to imply that performing arithmetic-type things would be valid? Dunno... Not too worried.

File: islandora_usage_stats.module (hunk 1 above)
Comment (willtp87) [inline islandora_usage_stats.module:321]: If I have time I'll take a look in the next pull.

File: views/islandora_usage_stats.views.inc (hunk 2 above)
Comment (willtp87) [inline views/islandora_usage_stats.views.inc:198]: Talked with Adam, gonna let this go through.

File: views/islandora_usage_stats.views_default.inc [hunk 3]
```diff
@@ -147,10 +179,7 @@ function islandora_usage_stats_views_default_views() {
   $handler->display->display_options['filters']['time']['date_fields'] = array(
     'islandora_usage_stats_object_access_log.time' => 'islandora_usage_stats_object_access_log.time',
   );
-
-  /* Display: Page */
-  $handler = $view->new_display('page', 'Page', 'page');
-  $handler->display->display_options['path'] = 'admin/reports/islandora_usage_stats';
+  $handler->display->display_options['path'] = 'islandora-usage-stats-report';
```
Comment (adam-vessey) [inline views/islandora_usage_stats.views_default.inc]: Currently rendering on the admin menu alone... Should it be under "Reports" or sommat?

**Pull request merged**